README.md

```diff
@@ -20,10 +21,8 @@ testing code that uses [jsverify](https://jsverify.github.io/) in
 What is a good invariant for your recursive implementation of `fib()`
 i.e. something that is always true at the beginning of the recursive call?
 
-Hint: Think about what the "state of the world" is here and what you can say
-about it at the start of each recursive call. Your invariant must say something
-about the current recursive call.
+At the beginning of `fibr(n, arr)`, the value of `n` is always less than the `n` of the parent function call (i.e. the function call, a level of recursion above, which called itself). Since `n` is decreasing with each recursive call and our terminating base case is `n < 2`, this helps proves the function will terminate. 
 
-Describe your reasoning and the conclusion you've come to. Your reasoning is the
-most important part. You do not need to prove that the invariant is correct. Add
-your answer to this markdown file.
+Additionally, in each recursive call, the algorithm will input the sum of the previous two array elements in the correct location in the array. This is the recursive invariant which helps inductively prove the correctness of the implmentation. At each recursive call, `fib[n] = fib[n - 1] + fib [n - 2]` or the base cases are used to insert the nessecary values.
```

Comment:
...and what must be true in order for the sum of the previous two elements to be the next Fibonacci number?

Reply:
Each element in the array (as the recursion "trickles up") is a correct Fibonacci number and appears in the correct position (ie. the i-th element is the i-th Fibonacci number).